applications/sink/router-sink/src/main/java/org/springframework/cloud/stream/app/sink/router/RouterSinkProperties.java

```diff
@@ -74,10 +75,10 @@ public class RouterSinkProperties {
 	/**
 	 * Where to send un-routable messages.
 	 */
-	private String defaultOutputChannel = "nullChannel";
```

Comment:
Starting with this version we cannot talk about `MessageChannel` abstraction. 
And since there is no `/dev/null` binding conversion in Spring Cloud Stream I find an empty value as sensible default and leave end-user to his/her decision what is such a `defaultOutputBinding` can be.

Reply:
Yeh, that makes total sense. Only concern is if some users are expecting / relying on this in current code. 

Maybe we could start a "Breaking changes" doc so that when we release we have these things captured. I know Spring Integration / Spring Kafka does a solid job of telling users what has changed/breaking etc..

Reply:
Sure! That's not bad idea.
I guess @sobychacko has already some experience in such a guidance for this project.

Reply:
I will just peek at what SI/SK is doing and start a doc up w/ this info in it.

Reply:
For that goal you can look here: https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-5.x-to-6.0-Migration-Guide

Reply:
I created this to follow up: https://github.com/spring-cloud/stream-applications/issues/426